testing/internal-integ-testing/src/main/groovy/org/gradle/integtests/fixtures/problems/ReceivedProblem.groovy

```diff
@@ -88,6 +88,10 @@ class ReceivedProblem implements Problem {
         definition
     }
 
+    Severity getSeverity() {
```

Comment:
We didn't have this before? Interesting...

Reply:
Nope, I was also surprised.